docs/content-crag/concepts/solids-pipelines/jobs.mdx

```diff
@@ -0,0 +1,156 @@
+---
+title: Jobs | Dagster
+description: A job is an executable particularization of a graph that binds it to resources and configuration.
+---
+
+# Jobs
+
+A job is an executable particularization of a graph that binds it to resources and configuration.
```

Comment:
I feel like we can probably come up something more concise than `particularization`

Reply:
Lol extremely true.  Replacing with "A job is a graph plus a set of resources and configuration."